main.cpp

```diff
@@ -133,30 +199,44 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 int main()
 {
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
-    auto carRented = rentACar(6, 2); 
-    
-    //1)
-    
-    //2)
-    
-    //3)
+    // auto carRented = rentACar(6, 2); 
     
+    //1)
+    int signalArray[] = {1, 2, 3};
+    sendSignal(signalArray, 3);
+    //2)
+    char dog[] = "dog";
+    auto log = changeFirstLetter(dog, 'l');
+    //3)
+    int toppings[] = {1, 2, 3};
+    auto orderID = placePizzaOrder(1, toppings);

```

Comment:
no arrays yet (especially not raw C-style arrays. this is C++, remember?)
they haven't been covered, nor have the gotchas been explained for them.
- [ ] revise